Listenbrainz/UI/Components/SettingsView.swift

```diff
@@ -39,7 +39,13 @@ struct SettingsView: View {
       .navigationBarItems(
         trailing: Button(action: {
           homeViewModel.requestMusicData(userName: userName)
-          feedViewModel.fetchFeedEvents(username: userName, userToken: userToken)
+          Task {
```

Comment:
This should not be needed here. Load the feed in the feed component only

Reply:
Works! @akshaaatt